scanners/amass/parser/e2e/helpers.js

```diff
@@ -0,0 +1,280 @@
+// SPDX-FileCopyrightText: 2020 iteratec GmbH
```

Comment:
The helper file is just here temporarily for testing purposes?
Duplicating it for every scanner & hook sounds tedious.

Reply:
Just for easy decoupled testing :+1: